source/blog/2018-05-11-the-emberjs-times-issue-46.md

```diff
@@ -57,15 +43,32 @@ App.AboutRoute = Ember.Route.extend({
     {{/each}}
   </ul>
 </script>
+```
 
-Using Ember-CLI instead has been recommended for years, but anyone still using this api should
+Using Ember-CLI instead has been recommended for years, but anyone still using this API should
 read the RFC and comment on why they might still need it.
 
+---
+
+## [EMBER CLI THINGS](#your-url-here)
+
 
 ---
 
-## [SOMETHING ELSE HERE](#your-url-here)
+## [Get involved with the new Ember guides](https://emberguides.stonecircle.io/release/)
+The Learning Team has been working on a new and improved version of the Guides app. It's now made with Ember.js and is more shiny than ever. The Guides app is almost ready for a proper release, but before we do so, we would love for you to [try it out](https://emberguides.stonecircle.io/release/). If you find any issues then please [report them](https://github.com/ember-learn/guides-app/issues).
+
+Or if you feel in a particulary good mood, look through existing issues in the [Guides app](https://github.com/ember-learn/guides-app/issues) or the [Guides source](https://github.com/ember-learn/guides-source/issues) and start contributing to the new guides.
+
+---
+
+## [Moar #EmberJS2018 blog posts wanted!](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html)
```

Comment:
I think sticking with "More" might look more professional, but maybe we don't care and want to look cool? 🤓

Reply:
I'm all for "moar" and being 😎 👍

Reply:
Either way! Was just trying to make it more interesting, since we mentioned it last week. 

I considered finding an [Uncle Sam "I want you" logo](https://techcrunch.com/wp-content/uploads/2015/04/uncle-sam-we-want-you1-kopie_1.png?w=1390&crop=1), really we need a Tomster like this 😅